ModeratorFrontEnd/AIForOrcas/AIForOrcas.Server/Controllers/DetectionsController.cs

```diff
@@ -536,23 +550,25 @@ public async Task<IActionResult> GetUnknownsAsync([FromQuery] DetectionQueryPara
 				// total number of records
 				double recordCount = queryable.Count();
 
+				var results = queryable
+					.Select(x => DetectionProcessors.ToDetection(x)).ToList();
```

Comment:
Is ToList necessary here? (see relevant comments in DetectionProcessors.cs below)

Reply:
Yes. See conversation above.